pages/index.js

```diff
@@ -76,9 +72,9 @@ export default function Home() {
         })
       );
```

Comment:
I think you should use lowercase only or uppercase only strings for comparison. Otherwise you might get weird results when comparing (`a` would come after `Z` for example)

Reply:
done. thanks for the hint